src/Controller/IndexController.php

```diff
@@ -95,7 +95,9 @@ public function searchAction()
         $settings = $this->page->settings();
         foreach ($settings['facets'] as $facet) {
             $query->addFacetField($facet['name']);
+            $query->addFacetSort($facet['name'], $facet['sort_by']);
```

Comment:
`$facet['sort_by']` may be undefined, and this will trigger a warning.
Or it may be empty, and in this case I think there is no need to call addFacetSort